ebpf-shared/src/lib.rs

```diff
@@ -3,6 +3,7 @@
 #[repr(C)]
 #[derive(Clone, Copy, Debug, PartialEq, Eq)]
 pub struct Signal {
+    pub cgroupid: u64,
     pub signr: i32,
```

Comment:
`signal`?

Reply:
I'll actually call this `signum`, I would like with those "shared" types to align as closely as possible with kernel code to be explicit what the fields are.

Reply:
I agree. If the field name is being taken from the kernel, stick to that name (we try to do the same with cgroups).